checkov/arm/utils.py

```diff
@@ -60,3 +60,21 @@ def get_files_definitions(
             parsing_errors.append(os.path.normpath(file))
 
     return definitions, definitions_raw, parsing_errors
+
+
+def extract_resource_name_from_resource_id_func(resource_id: str) -> str:
+    # Extract name from resourceId function
```

Comment:
this comment is exactly the function name, please explain and give an example of the format you are based on

Reply:
Done

Reply:
Done